code/game/objects/items/weapons/material/material_weapons.dm

```diff
@@ -92,7 +93,7 @@
 	if(health<=0)
 		if(fragile)
 			shatter(consumed)
-		else if(!dulled)
+		else if(!dulled && can_dull)

```

Comment:
If the only thing you're putting this on is the snowball, it's pointless. The fragile check should just destroy the snowball. If it doesn't that's bugged.

Reply:
Thought it would be useful for future weapons as well. Maybe just a variable to turn on if you want your weapon un-dullable? I can take it out, if it's too much of a problem.

Reply:
Nah, I will have use for it, but if the snowball is fragile, it doesn't really need not to dull.

Reply:
Then I just left it in for future use and took out the can_dull on the snowball.